playbook-website/app/javascript/components/MainSidebar/NavComponents/KitsNavComponent.tsx

```diff
@@ -160,24 +160,18 @@ export const KitsNavItem = ({
         text={linkFormat(categoryKey)}
       >
         {sublinks.map((sublink, j) => (
-          <>
-            {
-              sublink !== "advanced_table" && (
-                <NavItem
-                  active={calculateIsActiveCategory(j, null, sublink)}
-                  cursor="pointer"
-                  dark={dark}
-                  fontSize="small"
-                  key={`${sublink}-${j}`}
-                  link={generateLink(categoryKey, sublink, type)}
-                  marginY="none"
-                  onClick={() => handleSubItemClick(j, sublink, kitIndex)}
-                  paddingY="xxs"
-                  text={linkFormat(sublink)}
-                />
-              )
-            }
-          </>
+          <NavItem
+            active={calculateIsActiveCategory(j, null, sublink)}
+            cursor="pointer"
+            dark={dark}
+            fontSize="small"
+            key={`${sublink}-${j}`}
+            link={generateLink(categoryKey, sublink, type)}
+            marginY="none"
+            onClick={() => handleSubItemClick(j, sublink, kitIndex)}
+            paddingY="xxs"
+            text={linkFormat(sublink)}
+          />
```

Comment:
this is just removing the 'hack' we put into place [on this commit](https://github.com/powerhome/playbook/pull/3084/commits/309edb686610fc190012be7beb1feb8a3c592608) on PBNTR-177